infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/KubernetesPluginsToolingApplier.java

```diff
@@ -171,6 +176,15 @@ private void populateWorkspaceEnvVars(
         .collect(Collectors.toList());
   }
 
+  private K8sContainerResolver toK8sContainerResolver(
+      CheContainer container, List<ChePluginEndpoint> endpoints) throws InfrastructureException {
```

Comment:
`InfrastructureException` can not be thrown here

Reply:
Thanks, fixed